### PR TITLE
Add switch info to Uptimerobot

### PR DESCRIPTION
**Pull request by chemelli74**
## Proposed change

Add switch info to Uptimerobot

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/65394
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_integrations/uptimerobot.markdown [hunk 1]
```diff
@@ -23,4 +24,6 @@ The `uptimerobot` integration provides entities to get the status for all of you
 
 To get your API key, go to [My Settings](https://uptimerobot.com/dashboard#mySettings) on the UptimeRobot website, at the bottom you will find your "Read-Only API Key".
 
+**NOTE**: If you want to be able to use the active monitor switch then you need to find your "Main API Key".
+
```
Comment (ludeeus) [inline source/_integrations/uptimerobot.markdown]:
```suggestion
```
Change the line above to "Main API Key" instead.

**Review by frenck — DISMISSED** [review]
@chemelli74 Could you take a look at the last comment by Ludeeus?

Thanks! 👍 

../Frenck

File: source/_integrations/uptimerobot.markdown (hunk 1 above)
Comment (chemelli74) [reply source/_integrations/uptimerobot.markdown]: Done.
Due to the changed logic in the integration, your request makes sense.

**Comment by chemelli74** [comment]
> @chemelli74 Could you take a look at the last comment by Ludeeus?

Done

> Thanks! 👍

YW

Simone

**Review by ludeeus — APPROVED** [review]

**Pull request merged**